components/Tabs/index.jsx

```diff
@@ -9,11 +9,13 @@ export default class Tabs extends Component {
   static propTypes = {
     hash: PropTypes.string,
     items: PropTypes.arrayOf(PropTypes.element).isRequired,
-    pathname: PropTypes.string.isRequired
+    pathname: PropTypes.string.isRequired,
+    theme: PropTypes.oneOf(['default', 'kununuNav'])
```

Comment:
i wouldn't use any keyword from css to call the theme.
Twitter Bootstrap uses calls something similiar ```nav-justified```. How about ```tabbed```?